sandbox/removeRows.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+
+data = pd.read_csv(
+    "/usr/local/google/home/bumaschny/synthea/src/main/resources/providers/hospitals.csv"
+)
+
+print(len(data.index))
```

Comment:
do we need to print to stdout?
If we do, maybe append some description of what is to be printed at the beginning. For example:
`print(f"Num of rows: {len(data.index)}")`